test/signalrclient-e2e-tests/test_utils.h

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "stdafx.h"
+#include "cpprest\details\basic_types.h"
+
+utility::string_t get_url(int argc, utility::char_t* argv[]);
+
+utility::string_t url;
```

Comment:
It's about not having the variable in the header and not about not having the variable at all. (and if you define it in cpp then the issue below is solved because I think you will have to have `extern`s)